test/test_functional_tensor.py

```diff
@@ -593,18 +594,18 @@ def test_affine(self):
         with self.assertWarnsRegex(UserWarning, r"Argument resample is deprecated and will be removed"):
             res1 = F.affine(tensor, 45, translate=[0, 0], scale=1.0, shear=[0.0, 0.0], resample=2)
             res2 = F.affine(tensor, 45, translate=[0, 0], scale=1.0, shear=[0.0, 0.0], interpolation=BILINEAR)
-            self.assertTrue(res1.equal(res2))
+            assert_equal(res1, res2)
 
         # assert changed type warning
         with self.assertWarnsRegex(UserWarning, r"Argument interpolation should be of type InterpolationMode"):
             res1 = F.affine(tensor, 45, translate=[0, 0], scale=1.0, shear=[0.0, 0.0], interpolation=2)
             res2 = F.affine(tensor, 45, translate=[0, 0], scale=1.0, shear=[0.0, 0.0], interpolation=BILINEAR)
-            self.assertTrue(res1.equal(res2))
+            assert_equal(res1, res2)
 
         with self.assertWarnsRegex(UserWarning, r"Argument fillcolor is deprecated and will be removed"):
             res1 = F.affine(pil_img, 45, translate=[0, 0], scale=1.0, shear=[0.0, 0.0], fillcolor=10)
             res2 = F.affine(pil_img, 45, translate=[0, 0], scale=1.0, shear=[0.0, 0.0], fill=10)
-            self.assertEqual(res1, res2)
+            assert res1 == res2
```

Comment:
Are `res(1|2)` scalars here? If yes, shouldn't the same also apply to all the other cases above?

Reply:
They're PIL images, where `==` comparison is possible, while the rest of the cases above are tensors.
 
We can't use `assert_equal` on PIL images as we can't construct a tensor directly from those. I'll use `np.as_array` so that we still rely on `assert_equal` for consistency.